task/buildah-dance/0.1/buildah-dance.yaml

```
@@ -0,0 +1,253 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  labels:
+    app.kubernetes.io/version: "0.1"
+    build.appstudio.redhat.com/build_type: "docker"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: "image-build, appstudio, hacbs"
+  name: buildah-dance
```

```suggestion
  name: buildah-rhtap
```

and we can change the directory/filename too.

---

Changed it.